lib/tests/streamlit/runtime/scriptrunner/script_runner_test.py

```diff
@@ -1033,7 +1033,7 @@ def test_404_hash_not_found(self):
         self.assertEqual(
             scriptrunner._main_script_path,
             sys.modules["__main__"].__file__,
-            (" ScriptRunner should set the __main__.__file__" "attribute correctly"),
+            (" ScriptRunner should set the __main__.__file__attribute correctly"),
```

Comment:
nit: should be: `__main__.__file__ attribute`

Reply:
Hmhm are you sure? The equal tests passes and I guess the version right now is equivalent to the one before

Reply:
Ah, that's just the message 👍  Updated it!